include/ppx/xr_component.h

```diff
@@ -31,11 +31,15 @@
 #include <android_native_app_glue.h>
 #endif // defined(PPX_ANDROID)
 
+#include <queue>
```

Comment:
nit: should this be moved to line 40 below?

Reply:
Done.